BrainPortal/app/controllers/signups_controller.rb

```diff
@@ -166,13 +166,15 @@ def index #:nodoc:
 
     scope_default_order(@scope, 'created_at', :desc)
 
-    @base_scope       = Signup
-    @view_scope       = @scope.apply(@base_scope)
+    view_hidden                 = params[:view_hidden].presence == "true" ? true : false
```

Comment:
I was saying there's never any reason to have

```ruby
    boolean_expression ? true : false
```

in any piece of code, it's redundant.